crypto/BRCryptoWalletManager.c

```diff
@@ -1233,6 +1309,86 @@ cryptoWalletManagerSubmit (BRCryptoWalletManager cwm,
     }
 }
 
+extern void
```

Comment:
General pattern is inline with current BWM and EWM. Small departure in that every callback would be expected to call into the associated announce function, even on failure, so that the record could be free'd up.